pkg/assembler/backends/neo4j/backend.go

```diff
@@ -59,7 +59,7 @@ func GetBackend(args backends.BackendArgs) (backends.Backend, error) {
 		return nil, err
 	}
 	client := &neo4jClient{driver}
-	if config.TestData {
+	/* if config.TestData {
```

Comment:
@mihaimaruseac still using this? or we okay to comment it out to just what's been implemented?

Reply:
I will make a PR to clean this all up, just wanted to move all the testdata over to `cmd/graphql_playground/cmd/ingest.go` before we did.

Reply:
>This conflicts with the cmd/graphql_playground/cmd/ingest.go where more testdata is being added via the proper ingestion method. The commented-out code uses the old storeGraph that will all be removed.

I added a `GetEmptyBackend` function that should work? maybe?

Reply:
Add another `GetEmptyBackend` for neo4j? It would never be used. We are not losing anything by commenting out the above. It can be removed now as the data being ingested is not in the proper format via `storeGraph `

Reply:
I'd say we still need this for a while since we haven't ingested packages with the client queries.

However, if you get an error on ingesting from this one, then let's remove

Reply:
Keep it commented out for now or remove it? Its not causing an error, its just bad data being added on top off the good test data.

Reply:
Let's keep it commented, I might need the old ingested data for testing